src/components/atoms/AdminHeader/AdminHeader.tsx

```diff
@@ -47,12 +48,12 @@ export const AdminHeader: React.FC<HeaderProps> = ({
                 {crumbtrail.map((item) => (
                   <li key={item.name}>
                     <div className="flex items-center">
-                      <a
-                        href={item.href}
+                      <Link
+                        to={item.href}
                         className="text-sm font-medium text-gray-500 hover:text-gray-700"
                       >
                         {item.name}
-                      </a>
+                      </Link>
                       <svg
```

Comment:
[NOTE]

I think @denisdimitrov had an open question about embedding `svg`s.

We should probably talk about putting them as separate files in the assets directory.

Reply:
> We should probably talk about putting them as separate files in the assets directory.

yeah, sounds like a good idea. Haven't moved them yet. Can do it in the next PR.